action.yml

```diff
@@ -18,7 +18,7 @@ inputs:
     default: ""
 outputs:
   changed:
-    description: "JSON encoded object that contains the changed files. '{ changed: Array<string> }'"
+    description: "JSON encoded array that contains the changed files. 'Array<string>'"
```

Comment:
Is this just a docs change?  Or did the structure actually change?

Reply:
Just docs